src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -190,6 +191,38 @@ function ManufacturerLandingPage() {
               {manufacturerData.address.country}
             </Typography>
           </Grid>
+
+          <Grid item xs={12} my={2}>
+            <Typography textAlign={'center'} variant="h6">
+              Last modified:
+            </Typography>
+          </Grid>
+          <Grid item xs={12}>
+            <Typography
+              sx={{ mx: '8px' }}
+              textAlign={'center'}
+              variant="body1"
+              color="text.secondary"
+            >
+              {formatDateTimeStrings(manufacturerData.modified_time ?? '')}
```

Comment:
```suggestion
              {formatDateTimeStrings(manufacturerData.modified_time)}
```
If the types are changed, this shouldn't be needed. Would probably need to ctrl f on `formatDateTimeStrings(` to update any others as well.